### PR TITLE
Add ProtoOS logs E2E coverage

### PR DESCRIPTION
**Pull request by edgars-avotins**
## What changed
- add ProtoOS Logs E2E coverage for search, error/warning filters, no-results handling, and export downloads
- add a small ProtoOS Logs page object to keep the spec declarative
- add stable Logs test hooks for rows, empty state, and filter controls

## Why
- ProtoOS Logs previously had no page-specific E2E coverage beyond being a navigation waypoint in sleep/wake tests
- these scenarios are deterministic and simulator-friendly, which makes them a good first CI slice for ProtoOS Logs

## Validation
- npx eslint e2eTests/protoOS/pages/logs.ts e2eTests/protoOS/spec/logs.spec.ts src/protoOS/pages/MinerLogs/Logs.tsx
- npx playwright test spec/00-onboarding.spec.ts --project=desktop --grep "Complete onboarding flow" --no-deps
- npx playwright test spec/logs.spec.ts --project=desktop --no-deps
- npx playwright test spec/logs.spec.ts --project=mobile --no-deps

### CODE REVIEW

File: client/src/protoOS/pages/MinerLogs/Logs.tsx [hunk 1]
```diff
@@ -207,36 +207,40 @@ const Logs = ({ logsData, fetchMaxLogs }: LogsProps) => {
...
-                      "border-transparent bg-intent-warning-10": filterByLogType?.includes(logTypes.warn),
-                    },
-                    {
-                      "border-intent-warning-10": !filterByLogType?.includes(logTypes.warn),
-                    },
-                  )}
-                  selected={filterByLogType?.includes(logTypes.warn) || false}
-                  onClick={createToggleFilter(logTypes.warn)}
-                />
+                <div data-testid="logs-error-filter">
```
Comment (mcharles-square) [inline client/src/protoOS/pages/MinerLogs/Logs.tsx]: can the testid be set on `LogBadges` instead of introducing a new div here?

**Review by mcharles-square — APPROVED** [review]
